src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CaseViewFieldBuilder.java

```diff
@@ -1,14 +1,13 @@
 package uk.gov.hmcts.ccd.domain.model.aggregated;
 
-import uk.gov.hmcts.ccd.domain.model.definition.CaseEventFieldDefinition;
-import uk.gov.hmcts.ccd.domain.model.definition.CaseFieldDefinition;
```

Comment:
Could you please move  the imports and take developed  version ?

Reply:
reverted to develop version